website/content/blog/2024-10-09-meeting.md

```diff
@@ -0,0 +1,81 @@
+---
+
+## Attendee's
+
+- Dean Ellis - President
+- Simon Jackson - Corporate Secretary
+- Tom Spilman - Banking Matters
+- Marko Jeremic - Board Chairman
+- Thomas Altenburger - Board Member
+
+## Agenda
+
+- Developer Interviews
+- Patreon Discord Private Channels / General Rewards.
+- Need for new promo art/banners.
+- Possibility of a "regular" open hours / MonoGame chat.
+- Bounty updates
+
+## Developer Interviews
+
+To help promote and educate the developer community, we are seeking to interview Teams and Team members of titles that are actively building with MonoGame with the aim to build further support material to help people make games using real-world experiences.
+
+We have one developer lined up already and if anyone else would like to put their name forward, please email [biz@monogame.net](mailto://biz@monogame.net).
+
+### Patreon Improvements
+
+We have a great many sponsors on Patreon and we are looking to further our relationship with these supporters.  We have already rolled out a unique Discord Role/logo for Patreon's (which some have noticed already) and we are working on special rewards for higher tier members, ranging from:
+
+- Physical items
+- T-Shirts
+- Other rewards
+
+Such items would also be available later on a store for others to support MonoGame by purchasing.  The hardest part is getting good quality for a good price.
+
+We are also looking to link GitHub with Patreon through their integration to better highlight supporters there too.
+
+::: note Anonymous donations
+If you prefer to keep your donations private, then please let us know.  We will not publicly announce anyone's support without their approval.
+:::
+
+### New Promo art / banners
+
+To highlight the great games built with MonoGame, and to also highlight those titles too, we are looking to expand he showcase art we use in MonoGame promotion.  Initially we will add a new "media release" option to future showcase submissions, but we will be commissioning some newer artwork for the site. (and maybe merch?)
+
+### "Open Hours" (or AMA) is a go
+
+Following on from what Microsoft has been doing with their "Open Hours" weekly sessions, the MonoGame team have agreed to follow suit and we will be hosting a Monthly session (following the board meetings) to discuss MonoGame plans and open the floor up to questions.
+
+::: note First AMA meeting
+The first AMA meeting will be scheduled on Discord on Wednesday 16th October at 16:00 UTC (4PM UTC).
+Subsequent meetings will be at the same time one week after the regular MonoGame Foundation board meeting.
+:::
+
+More details to follow in a separate announcement.
+
+## Bounty updates
+
+Here is what we can share on the latest bounty updates:
+
+- The GDK/DX12 bounty is complete, but while this is usable on Xbox at the moment, it is recommended to wait until the Native implementation is complete.  This will also include a native implementation of DX 12 for windows.  Scheduled for MonoGame 4.0.
```

Comment:
This sentence doesn't really read well.  "it is recommended to wait until the Native implementation is complete." wait for what ? 

"it is recommended to wait until the Native implementation is complete before trying to use it commercially".?

Reply:
Updated